index.tsx

```diff
@@ -0,0 +1,137 @@
+import {memo, useEffect, useRef, useState} from "react";
+
+enum TimeStatus {
+    Minute,
+    Second,
+    MilliSecond
+}
+
+type TimerInfo = {
+    duration:number;
+    time:TimeStatus;
+}
+
+export const CodeReview = memo(()=>{
+    const [stopWatchTime, setStopWatchTime] = useState<number>(0)
+    const [isStopWatchRunning, setIsStopWatchRunning] = useState<boolean>(false)
+    const [timerTime, setTimerTime] = useState<number>(0)
+    const [isTimerRunning, setIsTimerRunning] = useState<boolean>(false)
```

Comment:
```suggestion
    const [stopWatchTime, setStopWatchTime] = useState(0)
    const [isStopWatchRunning, setIsStopWatchRunning] = useState(false)
    const [timerTime, setTimerTime] = useState(0)
    const [isTimerRunning, setIsTimerRunning] = useState(false)
```
state의 기본값 때문에 타입 추론이 되기 때문에 이렇게 생략해도 무방하답니다!